schema/config.json

```diff
@@ -0,0 +1,15 @@
+{
+  "title": "jupyterlab-nvdashboard",
+  "description": "Settings for the jupyterlab-nvdashboard extension.",
+  "type": "object",
+  "properties": {
+    "updateFrequency": {
+      "type": "integer",
+      "title": "Frequency of Updates",
+      "description": "The frequency of updates for the GPU Dashboard widgets, in milliseconds.",
+      "default": 1000,
```

Comment:
I think we should change the default to something like `250`.

From using this in practice lately a lot of my code was running in less than a second and wouldn't register at all on the graphs.

Reply:
There is seemingly no performance hit on either 100 or 250, do we want to go as low as possible? In that case 100ms would should be possible.